Celeste.Mod.mm/Mod/Everest/Everest.Content.cs

```diff
@@ -606,6 +611,11 @@ public static class Content {
                     Path.GetFileName(path).StartsWith("._"))
                     return false;
 
+                if(Loader.GlobalEverestIgnore.IsIgnored(path, metadata.Type == typeof(AssetTypeDirectory)) ||
+                    (metadata?.Source?.Ignore?.IsIgnored(path, metadata.Type == typeof(AssetTypeDirectory)) ?? false)) {
```

Comment:
```suggestion
                    (metadata.Source?.Ignore?.IsIgnored(path, metadata.Type == typeof(AssetTypeDirectory)) ?? false)) {
```
If `metadata` is `null` here, the first argument of the `||` has already crashed.